### ISSUE TITLE
ANN chapter

### ISSUE THREAD

**Issue by bpesquet** [bpesquet/ainotes#5]
- [x] Regularization
- [ ] Overfitting assessment using a validation set
